css/selectors.json

```diff
@@ -146,25 +130,20 @@
   ":active-view-transition": {
```

Comment:
Commenting for traceability: https://www.w3.org/TR/css-view-transitions-2/#the-active-view-transition-pseudo